### PR TITLE
ASIO description

### LINKED ISSUE #237: New Installer: Explain what ASIO is
The new installer is almost ready and linking to https://jamulus.io/wiki/Installation-for-Windows#setting-up-asio4all if no ASIO driver was found. We should explain ASIO on Windows for newbies. (Why they need it [only one sentence], and that ASIO4ALL is not recommended but probably needed. etc.)

Any proposals?
What I'd suggest:

1. Explain that you need an ASIO driver since the inbuilt drivers don't allow low latency audio
2. Tell the user that usually ASIO drivers are only provided for external sound cards (which we also recommend to use with Jamulus)
3. Tell the user that he can use ASIO4ALL which is a generic ASIO driver and the setup is described here (save for later); it should also be possible to download the .md file since sometimes the web browser also conflicts with ASIO4ALL

### PR DESCRIPTION
**Pull request by ann0see**
Since the new installer will link to this page, a description of ASIO is needed.
This paragraph looks a bit too long for me though.

### CODE REVIEW

File: wiki/en/en-Installation-for-Windows.md [hunk 1]
```diff
@@ -15,14 +15,19 @@ Make sure you read the [Getting Started](Getting-Started) page.
...
+*ASIO4ALL is not recommended if your sound card/audio interface provides its own ASIO driver, since that might have better latency.*
+
+For ASIO4ALL, you may need to experiment a bit to find the right configuration since every computer is a bit different.
+
+**Tip:** Setup your sound card while you're [connected to a server](Onboarding#2-connecting-to-a-server) to hear your instrument or voice and check if everything is correctly setup, but first read on.
 
 Before you start with Jamulus:
-1. **Close all applications** (especially those which could access your sound card like your browser/media player) since Jamulus needs exclusive access to your sound card.
-1. If the audio doesn’t work out of the box, make sure that only the **correct inputs/outputs** in ASIO4ALL **are switched on**. Everything else should be switched off. This can be done as follows:
+1. **Close all applications** (especially those which could access your sound card like your browser/media player) since Jamulus needs exclusive access to your sound card. Other programs will not be able to use audio if ASIO4ALL and Jamulus is running.
```
Comment (trebmuh) [inline wiki/en/en-Installation-for-Windows.md]: > if ASIO4ALL and Jamulus is running.

-> if ASIO4ALL and Jamulus **are** running.

**Comment by trebmuh** [comment]
I'm not a windows (hence ASIO) users so, I can't comment much on that front.
Looks good to me, nothing obviously wrong.

File: wiki/en/en-Installation-for-Windows.md (hunk 1 above)
Comment (ann0see) [reply wiki/en/en-Installation-for-Windows.md]: Done.

**Comment by ann0see** [comment]
I will merge this now and add a link on the released site to ASIO4ALL (this is needed since it might be possible that the new installer becomes live before the documentation is updated and translated).

**Pull request merged**